pitr/cli/internal/cmd/root.go

```diff
@@ -56,3 +63,99 @@ var RootCmd = &cobra.Command{
 		HiddenDefaultCmd:  true,
 	},
 }
+
+func getUserApproveInTerminal(prompt string) error {
+	fmt.Println(prompt)
```

Comment:
remove debug info

Reply:
i use logging.warn to replace it, this place is used to print some Message for user to check and need user input Y|N to continue.